portfolio/central/paths.py

```
@@ -25,13 +25,27 @@ def config_dir() -> Path:
-    if legacy.is_file():
+
+    def _has_sites(path: Path) -> bool:
+        if not path.is_file():
+            return False
+        try:
+            import json
+
+            raw = json.loads(path.read_text(encoding="utf-8"))
+            sites = raw.get("sites") if isinstance(raw, dict) else raw
+            return isinstance(sites, list) and len(sites) > 0
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)